packages/console/src/pages/TenantSettings/tabs/Domains/components/CustomDomain/components/ActiveProcess/components/DnsRecordsTable/index.module.scss

```diff
@@ -0,0 +1,48 @@
+@use '@/scss/underscore' as _;
+
+.tip {
+  font: var(--font-body-2);
+  color: var(--color-text-secondary);
+  margin-bottom: _.unit(2);
+}
+
+.container {
+  border-radius: 12px;
+  background-color: var(--color-layer-light);
+}
+
+.loading {
+  padding: _.unit(5);
+  display: flex;
+  align-items: center;
+  gap: _.unit(3);
+  font: var(--font-body-2);
+
+  .loadingIcon {
+    color: var(--color-primary);
+  }
+}
+
+.table {
+  padding: _.unit(2) 0;
+
+  .header {
+    background-color: unset;
+  }
+
+  .body {
+    background-color: unset;
+    padding-bottom: unset;
+
+    table > tbody > tr > td {
+      border: unset;
+    }
```

Comment:
table element nested under the .body?

Reply:
Sorry about the naming confusion. we use two <table /> in our `Table` component. one is used in the header, and another is used in the body. This name should be `bodyTableContainer`.

Also, the prop name in the `Table` is inaccurate.

```typescript
type Props = {
  headerClassName?: string;
  bodyClassName?: string;
}
```

It should be
```typescript
type Props = {
  headerTableClassName?: string;
  bodyTableContainerClassName?: string;
}
```

I will refactor it in LOG-6280.